blocks/init/src/Blocks/custom/column/manifest.json

```diff
@@ -89,6 +89,19 @@
 		},
 		"orderMobile": {
 			"type": "integer"
+		},
+
```

Comment:
Why the extra space?

Reply:
All item are grouped for responsive so it is easier to preview and manage

Reply:
Check this whole file

Reply:
Didn't notice the rest, GH collapses things. 👍🏼